source/includes/api/_dataset.md

```diff
@@ -545,7 +677,7 @@ curl -X POST https://api.resourcewatch.org/v1/dataset/find-by-ids \
 
 To create a dataset, you will need to be authenticated. Additionally, your user account needs to be associated with all `application` you specify for the dataset.
 
-To create a dataset, you need to define all of the required fields in the request body. The fields that compose a dataset are:
+You must provide all of the required fields in the request body. The fields that compose a dataset are:
```

Comment:
I would split this table into required attributes and optional attributes just so it's not so big and confusing. For attributes that are specific to different providers (`connectorUrl`, `tableName`, etc.), omit them and note in the `provider` description to see below for details on the different providers.